test-xtradb-operator/test/cr-haproxy-disabled.yaml

```diff
@@ -39,7 +39,7 @@ spec:
     gracePeriod: 600
   haproxy:
     enabled: false
-    size: 3
+    size: 1
```

Comment:
Just to confirm, the bug can still be reproduced even when the size is set to 1, is that correct?

Reply:
Indeed, the one haproxy size can also trigger the bug.

----


I modify this deliberately to reduce the workload waiting time (and also the waiting logic, otherwise you need wait on three pods).
Actually here I have some follow ups, it seems for this haproxy, if you set the size to be 3, and create them, then the third haproxy pod will be create in the last (e.g. `xtradb-haproxy-pod-3`). However, if you trigger the deletion, the three   haproxy pods will be deleted simultaneously and the `TERMINATED` status will just be **out of order**, which means you cannot simply wait for the `TERMINATED` for `xtradb-haproxy-pod-3`. So to simplify this, I just set the size to be 1.